lib/features/support/hooks.rb

```diff
@@ -1,13 +1,21 @@
 # frozen_string_literal: true
 
 require 'cucumber'
+require 'fileutils'
 require 'json'
 require 'securerandom'
 require 'selenium-webdriver'
 require 'uri'
 
 AfterConfiguration do |_cucumber_config|
 
+  # Clear out maze_output folder
+  maze_output = Dir.glob(File.join(Dir.pwd, 'maze_output', '*'))
+  unless maze_output.empty?
+    maze_output.each { |path| $logger.info "Clearing contents of #{path}" }
+  end
+  FileUtils.rm_rf(maze_output)
```

Comment:
Maybe this should be gated behind a `Maze.config.file_log` check? Or check if `maze_output` exists first?

Reply:
It should always have been in the guard above, but I think it would also be surprising if the folder got removed when running with `file_log` off - so have added that guard.